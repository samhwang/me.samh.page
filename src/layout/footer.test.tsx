import { render } from '@testing-library/react';
import { describe, it, expect } from 'vitest';
import Footer from './footer';
import RenderWrapper from '../test-utils/render-wrapper';

describe('Footer', () => {
  it('Should render without crashing and match snapshot', () => {
    const tree = render(<Footer />, { wrapper: RenderWrapper });
    expect(tree).toMatchSnapshot();
  });
});
