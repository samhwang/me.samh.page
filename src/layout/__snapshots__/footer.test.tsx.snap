// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer > Should render without crashing and match snapshot 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        class="resume-section p-3 p-lg-5 d-flex align-items-center"
      >
        <p>
          © 2022 
          <a
            href="https://github.com/samhwang"
          >
            Sam Huynh
          </a>
          .
           Hosted on 
          <a
            href="https://app.netlify.com"
          >
            Netlify
          </a>
          .
           Built with 
          <a
            href="https://www.vitejs.dev/"
          >
            Vite
          </a>
           and 
          <a
            href="https://reactjs.org/"
          >
            React
          </a>
          .
        </p>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      class="resume-section p-3 p-lg-5 d-flex align-items-center"
    >
      <p>
        © 2022 
        <a
          href="https://github.com/samhwang"
        >
          Sam Huynh
        </a>
        .
         Hosted on 
        <a
          href="https://app.netlify.com"
        >
          Netlify
        </a>
        .
         Built with 
        <a
          href="https://www.vitejs.dev/"
        >
          Vite
        </a>
         and 
        <a
          href="https://reactjs.org/"
        >
          React
        </a>
        .
      </p>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
